optaplanner-core/src/main/java/org/optaplanner/core/impl/partitionedsearch/scope/PartitionChangeMove.java

```diff
@@ -85,13 +86,13 @@ protected void doMoveOnGenuineVariables(ScoreDirector<Solution_> scoreDirector)
     }
 
     @Override
-    public boolean isMoveDoable(ScoreDirector<Solution_>  scoreDirector) {
+    public boolean isMoveDoable(ScoreDirector<Solution_> scoreDirector) {
         return true;
     }
 
     @Override
-    public PartitionChangeMove<Solution_> createUndoMove(ScoreDirector<Solution_> scoreDirector) {
-        throw new UnsupportedOperationException();
+    public NoChangeMove<Solution_> createUndoMove(ScoreDirector<Solution_> scoreDirector) {
+        return new NoChangeMove<>();
```

Comment:
We can't return NoChangeMove as that will not cause fail-fasts (but silent score corruption when used). Either we implement it or return null for now. Let's return null for now.